src/dotnetTestExplorer.ts

```diff
@@ -62,4 +59,13 @@ export class DotnetTestExplorer implements vscode.TreeDataProvider<vscode.TreeIt
         Executor.runInTerminal(`dotnet test --filter FullyQualifiedName~${methodName}`, this.cwd);
         AppInsightsClient.sendEvent("runTest");
     }
+
+    private evaluateWorkingDirectory(): void {

```

Comment:
Fair enough. So -- either there is a code solution that I don't know, or as I mentioned on the issues page it's a matter of informing users that the tests will run in their own terminal that the user shouldn't mess with.